### PR TITLE
DOC-107: Fix description of entity acceleraton property and deprecate

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Case: https://highfidelity.atlassian.net/browse/DOC-107

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21591/HighFidelity-Beta-PR16266-be3947f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21591/HighFidelity-Beta-Interface-PR16266-be3947f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21591/HighFidelity-Beta-PR16266-be3947f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21591/HighFidelity-Beta-Interface-PR16266-be3947f.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21591/HighFidelity-Beta-PR16266-be3947f.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21591/HighFidelity-Quest-Beta-PR16266-be3947f.apk)

File: libraries/entities/src/EntityItemProperties.cpp [hunk 1]
```diff
@@ -706,15 +706,11 @@ EntityPropertyFlags EntityItemProperties::getChangedProperties() const {
...
+ *     to an entity's motion only if its <code>dynamic</code> property is <code>true</code>.
  *     <p>If changing an entity's <code>gravity</code> from {@link Vec3(0)|Vec3.ZERO}, you need to give it a small 
  *     <code>velocity</code> in order to kick off physics simulation.</p>
- * @property {Vec3} acceleration=0,0,0 - A general acceleration in m/s<sup>2</sup> that the entity should move with, in world 
- *     coordinates. The acceleration is applied to an entity's motion only if its <code>dynamic</code> property is 
- *     <code>true</code>. The <code>acceleration</code> value is applied in addition to the <code>gravity</code> value.
- *     <p>If changing an entity's <code>acceleration</code> from {@link Vec3(0)|Vec3.ZERO}, you need to give it a small 
- *     <code>velocity</code> in order to kick off physics simulation.<p>
+ * @property {Vec3} acceleration - The current, measured acceleration of the entity, in m/s<sup>2</sup>.
+ *     <p class="important">Deprecated: This property is deprecated and will be removed.</p>
```
Comment (AndrewMeadows) [inline libraries/entities/src/EntityItemProperties.cpp]: It occurs to me: I think some entity types override what is the "acceleration" property.  `ParticleEffectEntityItem` comes to mind.  Anyone know?

Comment (AndrewMeadows) [reply libraries/entities/src/EntityItemProperties.cpp]: Meh, nevermind.  When I quickly scan exampl JS scripts I don't see any ParticleEffect logic that messes with the `acceleration` property directly.  Instead they mess with `accelerationSpread`.  So... full speed ahead.

**Pull request merged**